docs/design/queue/queue-priority-design.md

```diff
@@ -0,0 +1,418 @@
+# Volcano Priority For Queue
+
+@[zbbkeepgoing](https://github.com/zbbkeepgoing); Feb 10, 2023
+
+
+## Motivation
+
+Currently, Volcano has Job and Task level `PriorityClass` settings, but lacks Queue `PriorityClass` configuration.
+Queue's `PriorityClass` can play a role in the following scenarios.
+
+1. Submitters of tasks and jobs can't determine their `PriorityClass`, they only know to submit to one of the queues.
+2. The queue Priority capability can reduce the complexity of the job and the task submitter to a certain extent.
+   After the queue has priority, the task can properly adjust the priority according to its own task situation.
+3. The queue priority capability can help task that in high-priority queue to allocate resources first, which is very
```

Comment:
Do you mean it is used in the online jobs?

Reply:
I don't think this is for online business. This is related to the business form, depending on the user's usage scenario, online business may be only a part of it